class Input_Error(Exception):
    pass

class Zero_Error(Exception):
    pass